pagopa/io-app
# [IOBP-229] idpay code onboarding outcome screens

## Short description

THIS PR DEPENDS ON https://github.com/pagopa/io-app/pull/4976
addition of idpay code activation outcome screens
also all that is needed for navigation & co, and removal of the "CIE CODE" notation

<img width="250" alt="Screenshot 2023-09-13 at 17 38 13" src="https://github.com/pagopa/io-app/assets/60693085/b11ff758-5596-45f9-8574-d79815932632"><img width="250" alt="Screenshot 2023-09-13 at 17 38 33" src="https://github.com/pagopa/io-app/assets/60693085/71fafb1c-0875-43b5-9b72-99beae8dac4b">


## List of changes proposed in this pull request
- addition of said screen
- removal of IDPAY CIE notation
- addition of required I18n keys
- a big part of it all is mocked as you can see, will be updated in a future PR as soon as we get BE definitions
- comments where mocks were required

## How to test

navigate to profile>IDPAY code playground, then test both screens.
No back behaviour is expected, since it will not be present in the actual screens.
(sorry about that, you'll have to reload the app to test the other screen )
